### PR TITLE
DDF-2824 fixed install prereq caption & migration api titles

### PR DESCRIPTION
**Pull request by ricklarsen**
**PORT of PR https://github.com/codice/ddf/pull/2626**

#### What does this PR do?

- fixes table caption for Installation Prerequisites
- fixes title ordering in migration API docs

#### Who is reviewing it? 
@mcalcote @vinamartin @ahoffer @garrettfreibott 

#### Choose 2 committers to review/merge the PR. 

@clockard
@coyotesqrl
@lessarderic
@pklinef
@ricklarsen - Documentation
@rzwiefel
@shaundmorris

#### How should this be tested? (List steps with links to updated documentation)

- builds are successful. 
- content updated.

#### What are the relevant tickets?
[DDF-2824](https://codice.atlassian.net/browse/DDF-2824)

#### Screenshots (if appropriate)

html-format
pdf-format

#### Checklist:
- [x] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests
#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: distribution/docs/src/main/resources/content/_installing/installing-intro.adoc [hunk 1]
```diff
@@ -62,9 +60,12 @@ These are the system/environment requirements to configure _prior_ to an install
...
 
 |===
-*These requirements assume the use of a remote catalog. Extra resources will need to be allocated if the system will also be hosting a local catalog.
 
-**The amount of RAM can be increased to support memory-intensive applications. See <<jvm-memory-configuration, Memory Considerations>>
+^*^If installing a test, demo, or development installation, see the <<_quick_install_prerequisites,Quick Start Installation Prerequisites>>.
+
+^**^These requirements assume the use of a remote catalog. Extra resources will need to be allocated if the system will also be hosting a local catalog.
+
+^\***^The amount of RAM can be increased to support memory-intensive applications. See <<jvm-memory-configuration, Memory Considerations>>
```
Comment (mcalcote) [inline distribution/docs/src/main/resources/content/_installing/installing-intro.adoc]: 👻 I don't like the triple `*`, but I won't stand in your way.

**Comment by ricklarsen** [comment]
build now

File: distribution/docs/src/main/resources/content/_installing/installing-intro.adoc [hunk 2]
```diff
@@ -62,9 +60,15 @@ These are the system/environment requirements to configure _prior_ to an install
...
 |===
-*These requirements assume the use of a remote catalog. Extra resources will need to be allocated if the system will also be hosting a local catalog.
 
-**The amount of RAM can be increased to support memory-intensive applications. See <<jvm-memory-configuration, Memory Considerations>>
+[NOTE]
+====
+If installing a test, demo, or development installation, see the <<_quick_install_prerequisites,Quick Start Installation Prerequisites>>.
+====
+
+^*^These requirements assume the use of a remote catalog. Extra resources will need to be allocated if the system will also be hosting a local catalog.
```
Comment (ahoffer) [inline distribution/docs/src/main/resources/content/_installing/installing-intro.adoc]: What kind of resources? RAM? CPU? Disk space? More system administrators?

Comment (ricklarsen) [reply distribution/docs/src/main/resources/content/_installing/installing-intro.adoc]: All of the above?

Comment (ahoffer) [reply distribution/docs/src/main/resources/content/_installing/installing-intro.adoc]: I do not know. Who would know? Who  works on this part of the system?

Comment (ricklarsen) [reply distribution/docs/src/main/resources/content/_installing/installing-intro.adoc]: It would/could be any/all of the above.

**Comment by ricklarsen** [comment]
build now

**Pull request merged**